src/stores/login/index.ts

```diff
@@ -0,0 +1,55 @@
+import {makeAutoObservable} from 'mobx';
+import {propertyStore} from "../index";
+import {hash} from "@0auth/message";
+
+type LoginType = {
+  isLogin: boolean;
+  password: string;
+}
+
+class Login {
+  isLogin: boolean = false;
+  password: string = '';
+  hash: string = '';
+
+  constructor() {
+    makeAutoObservable(this)
+    this.init();
+  }
+
+  init() {
+    chrome.storage.local.get(['hash'], async data => {
+      const hash = data.hash;
+      if (hash === undefined) {
+        return;
+      }
+      this.hash = hash;
+    });
```

Comment:
Currently, it is difficult to check execution in the browser 
because this function can be called only for the chrome extension.